spring-cloud-dataflow-server/docker-compose.yml

```diff
@@ -95,7 +95,7 @@ services:
         echo 'Maven Task apps imported'"
 
   skipper-server:
-    image: springcloud/spring-cloud-skipper-server:${SKIPPER_VERSION:-2.7.0-SNAPSHOT}
```

Comment:
The `latest` is not a valid image tag nether for the SCDF or Skipper server nor for the Stream or Task apps.  An explicit app version tags is required.